src/include/packet_types.h

```diff
@@ -153,6 +153,14 @@ typedef struct altitude_data_block {
     uint8_t bytes[16];
 } TIGHTLY_PACKED AltitudeDataBlock;
 
-void altitude_data_block_init(AltitudeDataBlock *b, const int8_t measurment_type, const uint8_t pressure,
+void altitude_data_block_init(AltitudeDataBlock *b, const int8_t measurment_time, const uint8_t pressure,
```

Comment:
Could you correct this to `measurement_time`? It should also be unsigned.

Also, if you look at the spec for the altitude data block, each of these values should be 32 bits; right now you have them as eight.

Be careful with what you make signed (`int`) and unsigned (`uint`). For instance, temperature, altitude and pressure should be signed integers. The packet spec document will mentioned which fields should be signed.